tests/lib/rules/indent.js

```diff
@@ -405,28 +405,24 @@ ruleTester.run("indent", rule, {
             `,
             options: [4, { VariableDeclarator: 1, SwitchCase: 1 }]
         },
-        {
-            code: unIndent`
+        unIndent`
                 [{
                     foo: 1
                 }, {
                     foo: 2
                 }, {
                     foo: 3
                 }]
-            `
-        },
-        {
-            code: unIndent`
+            `,
```

Comment:
It would be nice to make the code example aligned with the `unIndent` call here:

```js
// currently:

unIndent`
        foo;
    `

// after:
unIndent`
    foo
`
```

(Unfortunately, `indent` doesn't check this because the template string actually contains spaces. However, the `unIndent` function ignores the spaces.)

However, it seems like manually changing everything would probably be very tedious. Separately, maybe we should add something to `lib/internal-rules` to enforce "indentation" of code samples inside template literals.